packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-attach/notebook-attach.html

```diff
@@ -0,0 +1,166 @@
+<div>
+    <!--Notebook attach object storage-->
+    <div class="mb-4">
+        <!--Title-->
+        <p class="oui-heading_5 mb-2">
+            <strong data-translate="pci_notebook_add_attach_object"></strong>
+        </p>
+
+        <!--Description-->
+        <span>
+            <p
+                class="mb-0"
+                data-translate="pci_notebook_add_attach_container_description"
+            ></p>
+            <a data-ui-sref="pci.projects.project.storages" target="_blank">
```

Comment:
Use a `ng-href` instead and have the link to `pci.projects.project.storages` generated in the route resolver

Reply:
Yes, already done